lib/bolt_server/transport_app.rb

```diff
@@ -184,6 +184,7 @@ def plugin_tarball(versioned_project, tarball_type)
             search_dirs << mod.pluginfacts if mod.pluginfacts?
             if tarball_type == 'all_plugins'
               search_dirs << mod.files if mod.files?
+              search_dirs << mod.scripts if mod.scripts?
```

Comment:
I'm not sure this is necessary. We don't do it [in the applicator](https://github.com/puppetlabs/bolt/blob/main/lib/bolt/applicator.rb#L241) and Puppet manifests shouldn't be using anything from the scripts directory.

Reply:
Ah, I think not having it in the Applicator is actually the bug. Users can use the `scripts` mount in Puppet catalogs, which works now if that bug is fixed. I think fixing that here makes sense.